examples/blueprint-construct/index.ts

```diff
@@ -286,5 +287,16 @@ export default class BlueprintConstruct {
             .teams(...teams, new blueprints.EmrEksTeam(dataTeam), new blueprints.BatchEksTeam(batchTeam))
             .enableControlPlaneLogTypes(blueprints.ControlPlaneLogType.API)
             .build(scope, blueprintID, props);
+
+        blueprints.CodePipelineStack.builder()
```

Comment:
This example blueprint is used for e2e test. As I see placeholder here for the connection arn I assume this will fail. 
This pipeline is a good candidate for a pattern (albeit requires release of the support). So a couple of things:

1. Please remove this code from the example as it will fail on e2e test.

2. Where you able to test this on your fork? If yes, then I can release, provided that you will add a pattern to the patterns repo which I will validate.

Reply:
Hi @shapirov103 ,
I have tested the addition using this pipeline builder, so why should I remove it? The only thing I changed was the "owner" connection to be my GitHub username.
This is how I validated that it is using the CodestarARN connection. 

Of course, I intend to modify the [GitOps & Pipeline pattern ](https://github.com/aws-samples/cdk-eks-blueprints-patterns/blob/main/lib/pipeline-multi-env-gitops/index.ts#L139) to use the CodeStar support once merged. 

I'll comment this out for now, as I see your point about that it cannot be tested unless changed manually.

Thanks